package.json

```diff
@@ -21,7 +21,8 @@
   "homepage": "https://github.com/peter-evans/create-or-update-comment#readme",
   "dependencies": {
     "@actions/core": "^1.1.1",
-    "@actions/github": "^2.0.0"
+    "@actions/github": "^2.0.0",
+    "ncc": "^0.3.6"
```

Comment:
I don't think this is necessary. Maybe a mistake? 😄

Reply:
You can run `npm run package` to kick ncc to build the dist directory if that was your intention.